packages/tailwind-config/tailwind.config.mjs

```diff
@@ -110,6 +111,9 @@ export default {
         '3xl': '0px 32px 64px -12px #1018283D',
         'input': '0px 0px 0px 4px #F2F2F224',
       },
+      borderColor: {
+        secondary: '#EBECEE'
```

Comment:
let's use the color that is already defined: `neutral-200`

Reply:
I remove this variable